src/fsharp/FSharp.Core/array.fs

```diff
@@ -57,7 +57,7 @@ namespace Microsoft.FSharp.Collections
         let create (count:int) (x:'T) =
             if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
             let array: 'T[] = Array.zeroCreateUnchecked count
-            for i = 0 to Operators.Checked.(-) count 1 do // use checked arithmetic here to satisfy FxCop
+            for i = 0 to Operators.Checked.(-) array.Length 1 do // use checked arithmetic here to satisfy FxCop
```

Comment:
I think you should do that in separate pr

Am 15.08.2016 15:16 schrieb "Jack Mott" <notifications@github.com>:

> In src/fsharp/FSharp.Core/array.fs
> <https://github.com/Microsoft/visualfsharp/pull/1420#discussion_r74759290>
> :
>
> > @@ -57,7 +57,7 @@ namespace Microsoft.FSharp.Collections
> >          let create (count:int) (x:'T) =
> >              if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
> >              let array: 'T[] = Array.zeroCreateUnchecked count
> > -            for i = 0 to Operators.Checked.(-) count 1 do // use checked arithmetic here to satisfy FxCop
> > +            for i = 0 to Operators.Checked.(-) array.Length 1 do // use checked arithmetic here to satisfy FxCop
>
> Just checked on another function, no performance impact there either. I'd
> be happy to get rid of all the Checked bits for readability purposes in
> this PR, or I could do another one. @KevinRansom
> <https://github.com/KevinRansom> or @dsyme <https://github.com/dsyme> any
> input here?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/visualfsharp/pull/1420/files/a177d8e52bcead89951983d8513dfa43a80c2bee#r74759290>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADgNJoBBWS3WMUlWScR4X6WE9HR4mGmks5qgGa4gaJpZM4Jh_aU>
> .
>